.github/workflows/test_branches.yml

```diff
@@ -376,9 +376,10 @@ jobs:
         echo "*** Install Pyomo dependencies ***"
         # For windows, cannot use newer setuptools because of APPSI compilation issues
         # There seems to be some specific problem with platformdirs 4.5.0
-        # on win 3.13/3.14 as of 2025-10-23
+        # on win 3.13/3.14 as of 2025-10-23 and platformdirs 4.5.1 on win/3.13
+        # as of 2025-12-12
         if test "${{matrix.TARGET}}" == 'win'; then
-            CONDA_DEPENDENCIES="$CONDA_DEPENDENCIES setuptools<74.0.0 platformdirs!=4.5.0"
+            CONDA_DEPENDENCIES="$CONDA_DEPENDENCIES setuptools<74.0.0 platformdirs<4.5.0"
```

Comment:
The only concern I have with this is we will never see if the issue is resolved (we will permanently be pinned before 4.5.0).  I think I would prefer:
```
platformdirs!=4.5.0,!=4.5.1
```
(that's legal for `pip`... does it also work for `conda`?)

Reply:
Good point. I tried making that change and we'll see if conda is happy with it.